### PR TITLE
chore: use warp route whitelist

### PR DESCRIPTION
**Pull request by ltyu**
Looks like this was incorrectly configured previously.

https://discord.com/channels/935678348330434570/1298686381698060441/1298702557148872784
"Part of the reason for the confusion is that the ezeth branch was setup wrong. Routes are supposed to live in the registry and then the warp UI just picks the ones it wants via the warpRouteWhitelist..."

### CODE REVIEW

File: src/consts/warpRoutes.yaml [hunk 1]
```diff
@@ -1,264 +1,6 @@
...
-    logoURI: /deployments/warp_routes/EZETH/RedETH-logo.svg
-    name: Restaked ETH on Sei
-    standard: EvmHypXERC20
-    symbol: redETH
+# A list of Warp Route token configs
+# These configs will be merged with the warp routes in the configured registry
+# The input here is typically the output of the Hyperlane CLI warp deploy command
+---
+tokens: []
+options: {}
```
Comment (paulbalaji) [inline src/consts/warpRoutes.yaml]: prettier is probably complaining about the lack of trailing line

File: src/consts/warpRouteWhitelist.ts [hunk 2]
```diff
@@ -2,8 +2,7 @@
 // Warp Route IDs use format `SYMBOL/chainname1-chainname2...` where chains are ordered alphabetically
 // If left null, all warp routes in the configured registry will be included
 // If set to a list (including an empty list), only the specified routes will be included
-export const warpRouteWhitelist: Array<string> | null = [];
-// Example:
-// [
-//   // 'ETH/ethereum-viction'
-// ];
+export const warpRouteWhitelist: Array<string> | null = [
+  'ezETH/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-taiko-zircuit',
```
Comment (jmrossy) [inline src/consts/warpRouteWhitelist.ts:6]: Just confirming, do we need to update the registry version on this branch of it's already good?

**Review by jmrossy — APPROVED** [review]
Just one question and like Paul said, you need to make prettier happy :)
Thanks!

File: src/consts/warpRouteWhitelist.ts (hunk 2 above)
Comment (ltyu) [reply src/consts/warpRouteWhitelist.ts:6]: its already good

**Pull request merged**